src/main/java/com/flightstats/hub/dao/file/FileUtil.java

```diff
@@ -70,9 +72,57 @@ static boolean write(byte[] bytes, String filename, String path) {
         return list;
     }
 
-    static boolean delete(String filepath) {
+    public static boolean delete(String filepath) {
         Path path = Paths.get(filepath);
         File file = path.toFile();
         return FileUtils.deleteQuietly(file);
     }
+
+    public static boolean directoryExists(String directoryPath) {
+        Path path = Paths.get(directoryPath);
+        return Files.exists(path);
+    }
+
+    public static void createDirectory(String directoryPath) throws IOException {
+        try {
+            Path path = Paths.get(directoryPath);
+            Files.createDirectories(path);
+        } catch (IOException e) {
+            logger.warn("unable to create directory {}", directoryPath);
+            throw e;
+        }
+    }
+
+    public static void mergeDirectories(String from, String to) throws IOException {
+        logger.info("merging {} into {}", from, to);
+
+        Path source = Paths.get(from).toAbsolutePath();
+        Path destination = Paths.get(to).toAbsolutePath();
+
+        List<Path> paths = Files.walk(source).collect(Collectors.toList());
+        for (int i = 0; i < paths.size(); ++i) {
+            Path path = paths.get(i);
+
+            if (!Files.exists(path)) {
+                logger.debug("skipping, already moved {}", path);
+                continue;
+            }
+
+            if (path.startsWith(destination)) {
```

Comment:
I'm not sure this is correct.

Reply:
Agreed. Closing this PR again until I can make sure its bulletproof or not needed at all :p